airflow/kubernetes/pod_launcher.py

```diff
@@ -108,6 +108,7 @@ def start_pod(
         curr_time = dt.now()
         if resp.status.start_time is None:
             while self.pod_not_started(pod):
+                self.log.warning("Pod not yet started: %s", pod.metadata.name)
```

Comment:
Should we remove log print on 116? 
<img width="676" alt="Screenshot 2020-06-28 at 18 15 36" src="https://user-images.githubusercontent.com/12058428/85952696-65df6580-b96b-11ea-8933-5729592a3ed9.png">